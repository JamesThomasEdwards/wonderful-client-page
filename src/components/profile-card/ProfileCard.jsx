// React
import React from 'react';

const ProfileCard = () => {

    return (
        <div className="profile-card">
            <div className="profile-card-top">
            </div>
            <div className="profile-card-bottom">
                <h2 className="profile-card-bottom__header">
                    Johanna Greene
                </h2>
                <p className="profile-card-bottom__text">
                    Product Designer
                </p>
                <p className="profile-card-bottom__text">
                    Wonderful
                </p>
                <p className="profile-card-bottom__text">
                    (808) 909-1101
                    </p>
                <p className="profile-card-bottom__text">
                    JGreene@gmail.com
                </p>
                <p className="profile-card-bottom__text">
                    http://www.Greenelikethecolor.com
                </p>
            </div>
        </div>
    );
};

export default ProfileCard;

