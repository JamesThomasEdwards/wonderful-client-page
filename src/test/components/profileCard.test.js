// React;
import React from 'react';
// Files;
import ProfileCard from '../../components/profile-card/ProfileCard.jsx';
// Testing
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render App component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<ProfileCard />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});