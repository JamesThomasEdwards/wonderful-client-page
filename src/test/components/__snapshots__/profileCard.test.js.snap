// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div
  className="profile-card"
>
  <div
    className="profile-card-top"
  />
  <div
    className="profile-card-bottom"
  >
    <h2
      className="profile-card-bottom__header"
    >
      Johanna Greene
    </h2>
    <p
      className="profile-card-bottom__text"
    >
      Product Designer
    </p>
    <p
      className="profile-card-bottom__text"
    >
      Wonderful
    </p>
    <p
      className="profile-card-bottom__text"
    >
      (808) 909-1101
    </p>
    <p
      className="profile-card-bottom__text"
    >
      JGreene@gmail.com
    </p>
    <p
      className="profile-card-bottom__text"
    >
      http://www.Greenelikethecolor.com
    </p>
  </div>
</div>
`;
